docs/concept-exercises.md

```diff
@@ -47,7 +47,7 @@ When designing Concept Exercises, please consider the following guidelines:
 
 ## Style Guidelines
 
-When writing documentation and supporting Markdown files for Concept Exercises, please consult the [style guide][style-guide.md] (and any language-specific style guides if applicable).
+When writing documentation and supporting Markdown files for Concept Exercises, please consult the [style guide][style-guide.md] and especially the [automatic formatting section][style-guide.md-auto-formatting]. Also check any language-specific style guides, where applicable.
```

Comment:
The whole thing is important. I think the fact that we draw attention to this specific bit is enough without saying "especially".

```suggestion
When writing documentation and supporting Markdown files for Concept Exercises, please consult the [style guide][style-guide.md] noting the [automatic formatting section][style-guide.md-auto-formatting]. Also check any language-specific style guides, where applicable.
```